packages/cfd/package.json

```diff
@@ -37,6 +37,10 @@
         "@types/react-dom": "^18.0.0",
         "babel-eslint": "^10.1.0",
         "babel-loader": "^8.1.0",
+        "@testing-library/jest-dom": "^5.12.0",
```

Comment:
I believe this is not needed as the base config is from the root level.
Note: All the imports here are optional, we should be able to run test using `npm run test:jest cfd`

Reply:
We are able to run `npm run test:jest cfd` even without this dependencies. Right now it works because of the way node_modules resolve dependencies so we have it in the root on the monorepo and it can find it to use it. But this is not safe, we should add the dependencies for each package separately. We can discuss it with Farzin, as he suggested this solution for trader package (when we worked on Turbos)